packages/ffe-cards-react/src/index.d.ts

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+
+export interface CardBaseProps extends React.HTMLAttributes {
+    children: React.ReactNode;
+    className?: string;
+    element?: React.ReactNode;
+    to?: string;
+}
+
+export interface IconCardProps extends CardBaseProps {
+    icon: React.ReactNode;
+    condensed?: boolean;
+    greyCharcoal?: boolean;
+}
+
+export interface ImageCardProps extends CardBaseProps {
+    image: React.ReactNode;
+}
+
+export interface ComponentBaseProps
+    extends React.HtmlHTMLAttributes<HTMLElement> {
+    className?: string;
+    element?: HTMLElement | string | React.ElementType;
+    children?: React.ReactNode;
+}
```

Comment:
```suggestion
export interface ComponentBaseProps
    extends React.HtmlHTMLAttributes<HTMLElement> {
    className?: string;
    element?: HTMLElement | string | React.ElementType;
    children?: React.ReactNode;
}
```

Reply:
I've been looking at this for a while but can't spot the difference here.

Reply:
I tried to push, but ended up creating a new branch. You can see all the changes to the file there; https://github.com/SpareBank1/designsystem/commit/ab4960626e6fdc40483b0034f2db2ac255336c3a

Reply:
Seems the only effective diff then ends up being the ordering of exports :)

Reply:
I don't have any strong opinions about that so I pushed it